CHANGELOG.md

```diff
@@ -1,4 +1,96 @@
-## 2.18.2 (Unreleased)
+## 2.19.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.4.2416**
+- Supported Terraform version: **v0.12.x**, **v0.13.x** and **v0.14.x**
+
+### Features:
+1. Implemented support of following attributes in **aviatrix_spoke_gateway**:
+  - ``enable_private_vpc_default_route``
+  - ``enable_skip_public_route_table_update``
+  - ``enable_auto_advertise_s2c_cidrs``
+2. Implemented support of associating a controller with a CoPilot instance:
+  - New resource **aviatrix_copilot_association**
+3. Implemented support for enabling event triggered HA for Site2Cloud type connection resources:
+  - New attribute ``enable_event_triggered_ha`` in **aviatrix_site2cloud**, **aviatrix_transit_external_device_conn**, **aviatrix_vgw_conn** and **aviatrix_device_transit_gateway_attachment**
+4. Implemented support for GCP FireNet with Fortinet and CheckPoint firewall vendors
+5. Implemented support for Insane Mode over Internet with following attributes in **aviatrix_transit_gateway_peering**:
```

Comment:
5. Implemented Insane Mode Support over Public Network for Transit Peering in **aviatrix_transit_gateway_peering**: